### DISCUSSION TITLE
REST API: How to change an option picker options by selecting another option picker？

### DISCUSSION THREAD

**Discussion by zhengtulymGh** [Budibase/budibase#5344] [Help]
There're two options picker，"Project Type Options Picker" and "Project Options Picker".  "Project Type Options Picker"  provides the the type options of "Project Options Picker".

I would like to known how to change the options of "Project Data Provider" by selecting "Project Type Options Picker".

Project Type:
<img width="1049" alt="v消息20220411-152321" src="https://user-images.githubusercontent.com/13768025/162685796-447919ba-2a1a-42d4-b03f-31c49bc361c6.png">
<img width="1609" alt="v消息20220411-152428(3)" src="https://user-images.githubusercontent.com/13768025/162685822-36af74c0-dca3-4c0b-a219-45ca11deb8b1.png">
<img width="1623" alt="v消息20220411-152428(4)" src="https://user-images.githubusercontent.com/13768025/162685824-bb1ba533-fdb8-40ac-811c-b0ac2e7aa2f2.png">

Project:
<img width="1035" alt="v消息20220411-152428(5)" src="https://user-images.githubusercontent.com/13768025/162685829-be24f5bb-0c28-421f-875b-901aef98f423.png">
<img width="1623" alt="v消息20220411-152428(1)" src="https://user-images.githubusercontent.com/13768025/162685813-e6bf1b2f-b1ac-4372-9976-bf6383ac91c8.png">
<img width="1605" alt="v消息20220411-152428(2)" src="https://user-images.githubusercontent.com/13768025/162685818-6d531a9d-87b4-4b06-a6a4-675dbed3d0b1.png">

DefineFilter not working for me:
<img width="1579" alt="售后系统级明细-202204111204" src="https://user-images.githubusercontent.com/13768025/162690716-44bc6d90-881a-4373-91f6-23f0055fec96.png">

**Comment by melohagan** [comment]
Hi @zhengtulymGh 

There had been a bug with query filters which was recently fixed. Could you make sure you are using the latest version by clicking the blue recycle circle near the top right.

If that doesn't help, this related discussion might: https://github.com/Budibase/budibase/discussions/4846

**Reply by zhengtulymGh** [reply]
I setup the project locally， and the package version is "1.0.105-alpha.9".
May be because number and string are not equal，such as '3' not equal to 3?
